.github/workflows/install-dependencies.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
-# Copyright 2020 Google LLC
+
+# Copyright 2021 Google LLC
```

Comment:
Nitpick (feel free to ignore):
Feel free to undo the changes to the copyright years (based on Olivier's point that we shouldn't touch these).
I'm good either way.